packages/swirl-components/src/components/swirl-color-input/swirl-color-input.tsx

```diff
@@ -35,6 +35,7 @@ export class SwirlColorInput {
   @Prop() pickerLabel?: string = "Color picker";
   @Prop() placeholder?: string;
   @Prop() required?: boolean;
+  @Prop() toolip?: string;
```

Comment:
Issue: The tooltip prop should be part of the form control component. No need to add it to every input component.

Reply:
That was some legacy code from testing. Thanks for the catch ;)

Reply:
Yes you are right